src/graphic-device/index.md

```diff
@@ -14,6 +14,4 @@ Used within our wordmark and as a graphic device across all GOV.UK channels, the
 <!-- TODO: this is not the final MP4 file
 but it's good to have one file to see and play around with the video player -->
 
-<video src="./dot-animations.mp4" controls width="720"></video>
-
```

Comment:
Was there a reason for removing this? I assume it was intended for the examples in the video.